index.html

```diff
@@ -12,412 +13,495 @@
+                            fill="#6070FF" />
+                    </svg>
+                </div>
+            </a>
+
+            <div class="nav-list">
+                <ul class="nav-list-item">
+                    <li>
+                        <a href="#portfolio">Portfolio</a>
+                        <a href="#about">About</a>
+                        <a href="#contact">Contact</a>
+                    </li>
+                </ul>
+            </div>
+
+            <div onclick="toggle()" class="hamburger">
+                <span class="bar"></span>
+                <span class="bar"></span>
+                <span class="bar"></span>
```

Comment:
You did an excellent job by implementing the disappearance of the menu on selecting any of the options.

Currently, when I scroll back to the hamburger menu after selecting an option on the mobile menu, I have to click twice in order for the menu to show up again. I suggest that you make it show up with a single click.

Reply:
@emmiiorji sir please the issue of the hamburger button is not happening on my side so I don't know how to go about it. could you please add more details on what is happening, or try to retry again?

Reply:
I see now that it works. 

Kindly ignore this comment and work on the disabled linters and you'll be good to go.